code/modules/jobs/job_types/security.dm

```diff
@@ -272,8 +272,71 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	else
 		to_chat(M, "<b>You have not been assigned to any department. Patrol the halls and help where needed.</b>")
 
+	grant_department_bonus(H, department)
 
 
+/datum/job/officer/grant_department_bonus(mob/living/carbon/human/H, department)
+	var/random = rand(1,3)
+
+	var/flavour_text
+
+	switch(department)
+		if(SEC_DEPT_SUPPLY)
+			switch(rand)
+				if(1)
+					var/gun_path = pick(/obj/item/gun/ballistic/shotgun/automatic/combat, /obj/item/gun/ballistic/shotgun/boltaction, /obj/item/gun/ballistic/automatic/wt550)
+					var/gun = new gun_path
+					H.put_in_hands(gun)
+					flavour_text = "For Close Encounters: Whether inspired by a bad experience with mining fauna, or simply from your connections with the Cargo Techs and their arms shipments, you've acquired an appreciation for classic solid projectile weapons. You carry one with you at all times."
+				if(2)
+					flavour_text "Contraband: Whether confinscated, or given as a bribe to make you look the other way, you are enjoying your own slice of Cargos more illicit side. You have a [contraband item]
+					var/contraband = (//traitor items?)
+				if(3)
+					flavour_text "Friend of Miners: Whether you've gone toe to toe against hostile natives, or the miners just pity a friendly face stuck on the station, they've taken a liking to you, and gifted you [mining_item]
+					var/mining_gift = pick(lavaland item? mining gear?)

```

Comment:
Try bone talisman and such, most other lavaland items and mining gear is extremely powerful.